migration/src/m0000300_create_product_version.rs

```diff
@@ -54,6 +54,7 @@ impl MigrationTrait for Migration {
                     .unique()
                     .col(ProductVersion::ProductId)
                     .col(ProductVersion::Version)
+                    .col(ProductVersion::SbomId)
```

Comment:
I think the consensus right now is to not modify the original migration, but to create a new one, altering the table structure. To me it looks like that migration already existed before this PR.

Reply:
Moved it to the new migration that drops and recreates the index

Reply:
But doesn't this PR still contain the additional column?

Reply:
Forgot to remove it 🤦